packages/react-instantsearch/package.json

```diff
@@ -33,10 +33,7 @@
     "algoliasearch-helper": "^2.21.0",
     "classnames": "^2.2.5",
     "lodash": "^4.17.4",
-    "prop-types": "^15.5.10",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-native": "^0.54.2"
+    "prop-types": "^15.5.10"
```

Comment:
Are prop types used in production builds?

Reply:
we can't strip them out in the library itself, since that way they'd only be useful for ourselves. People would strip them out at their bundling, not our I think

Reply:
In production build they are still present yes. But they are are used only in development. You can strip them with a babel plugin as @Haroenv suggest.